refinery/tool_manager/tests.py

```diff
@@ -2668,12 +2668,6 @@ def test_get_request_authenticated(self):
         self.create_tool(ToolDefinition.VISUALIZATION)
         self.assertIsNotNone(self.get_response)
 
-    def test_get_request_no_auth(self):
```

Comment:
We check for obj level perms in the API therefore we do not need the api level check. Units tests were already written for the obj level perms, so these are unnecessary now.